lib/common/output_xml.c

```diff
@@ -467,7 +467,7 @@ pcmk__output_xml_add_node_copy(pcmk__output_t *out, xmlNodePtr node) {
     // Shouldn't happen unless the caller popped priv->root
     CRM_CHECK(parent != NULL, return);
 
-    add_node_copy(parent, copy_xml(node));
+    add_node_copy(parent, node);
```

Comment:
All regression tests passed with this change. I still wonder if I used `copy_xml()` to fix some arcane XML doc issue when filing https://github.com/ClusterLabs/pacemaker/commit/b64368eb96a059ae62c414f5e373a78487829428, or if the `copy_xml()` is just a leftover that I forgot to remove. Obviously this commit assumes the latter.